src/components/visualizations/chart/legend/StaticLegend.tsx

```diff
@@ -1,4 +1,4 @@
-// (C) 2007-2019 GoodData Corporation
+// (C) 2007-2020 GoodData Corporation
```

Comment:
This file does not change anything, revert it, please.

Reply:
fixed

Reply:
fixed

Reply:
It has not been fixed

Reply:
My bad :(. It happens again after rebase. Fixed